test/support/fixtures/accounts_fixtures.ex

```diff
@@ -14,13 +14,9 @@ defmodule Demo.AccountsFixtures do
     user
   end
 
-  def capture_user_token(fun) do
-    captured =
-      ExUnit.CaptureLog.capture_log(fn ->
-        fun.(&"[TOKEN]#{&1}[TOKEN]")
-      end)
-
-    [_, token, _] = String.split(captured, "[TOKEN]")
+  def extract_user_token(fun) do
+    {:ok, captured} = fun.(&"[TOKEN]#{&1}[TOKEN]")
+    [_, token, _] = String.split(captured.body, "[TOKEN]")
     token
   end
 end
```

Comment:
Sorry for the trouble @aaronrenner but this is a new commit. We integrated Swoosh with the auth system in one of our apps and this lead to some improvements to the generated mailer code, which you can find in this commit. :)